### ISSUE TITLE
[Feature] 饼图aniamtionDuration设置动画时长无效问题

### ISSUE THREAD

**Issue by userxiong** [apache/echarts#17016]
### What problem does this feature solve?

Using the timeline attribute to control the animation duration can make the playback look smooth

### What does the proposed API look like?

var xAxisData = [];
var data1 = [];
var data2 = [];
for (var i = 0; i < 100; i++) {
  xAxisData.push('A' + i);
  data1.push((Math.sin(i / 5) * (i / 5 - 10) + i / 6) * 5);
  data2.push((Math.cos(i / 5) * (i / 5 - 10) + i / 6) * 5);
}
option = {
  title: {
    text: 'Bar Animation Delay'
  },
  legend: {
    data: ['bar', 'bar2']
  },
  toolbox: {
    // y: 'bottom',
    feature: {
      magicType: {
        type: ['stack']
      },
      dataView: {},
      saveAsImage: {
        pixelRatio: 2
      }
    }
  },
  tooltip: {},
  xAxis: {
    data: xAxisData,
    splitLine: {
      show: false
    }
  },
  yAxis: {},
  series: [
    {
      name: 'bar',
      type: 'pie',
      data: data1,
      emphasis: {
        focus: 'series'
      },
     
    },
    {
      name: 'bar2',
      type: 'bar',
      data: data2,
      emphasis: {
        focus: 'series'
      },
      
    }
  ],
  animationEasing: 'linear',
  
  animationDuration: 3000
};

**Comment by nelsonbbarriento23** [comment]
N 6w

**Comment by nelsonbbarriento23** [comment]
Come now babe

**Comment by mask2012** [comment]
yes, I have the same result, you can just open echarts.apache.org , select any pie chart demo, add
animation:true,
animationDuration: 1000*8,
and you will find it has no effect.
